pkg/services/librarypanels/database.go

```diff
@@ -62,3 +62,54 @@ func (lps *LibraryPanelService) deleteLibraryPanel(c *models.ReqContext, panelID
 
 	return err
 }
+
+// getLibraryPanels gets Library Panels based on statement and args
+func getLibraryPanels(session *sqlstore.DBSession, statement interface{}, args ...interface{}) ([]LibraryPanel, error) {
+	libraryPanels := make([]LibraryPanel, 0)
+
+	err := session.SQL(statement, args...).Find(&libraryPanels)
+
+	return libraryPanels, err
+}
+
+// getLibraryPanel gets a Library Panel
+func (lps *LibraryPanelService) getLibraryPanel(c *models.ReqContext, panelID int64) (LibraryPanel, error) {
+	orgID := c.SignedInUser.OrgId
+	libraryPanel := LibraryPanel{}
+
+	err := lps.SQLStore.WithDbSession(context.Background(), func(session *sqlstore.DBSession) error {
+		libraryPanels, err := getLibraryPanels(session, "SELECT * FROM library_panel WHERE id=? and org_id=?", panelID, orgID)
+
+		if err != nil {
```

Comment:
Can we just inline getLibraryPanels? That abstraction is so thin, I think it only makes it more difficult to read the code and see what actually happens. I'd find the following much easier to read:
```suggestion
		var libraryPanels []LibraryPanel
		err := session.SQL("SELECT * FROM library_panel WHERE id=? and org_id=?", panelID, orgID).Find(&libraryPanels)
		if err != nil {
```

Reply:
Sure, I can see your point. Will fix this in my next commit.

Reply:
Removed